staff/durack/publication.html

```diff
@@ -60,16 +60,16 @@ <h2>Publications:</h2>
 <b>Co-authored publications:</b><br>
 <br>
 
-Rieger, Landon A., Jason N. Cole, John C. Fyfe, Stephen Po-Chedley, Philip J. Cameron-Smith, <b>Paul J. Durack</b>, Nathan P. Gillett and Qi Tang (2020) Quantifying CanESM5 and EAMv1 sensitivities to volcanic forcing for the CMIP6 historical experiment, <i>Geoscientific Model Development</i>, <b>in review</b>. DOI: <a href="links/Riegeretal20GMD.html" target="_blank">10.5194/gmd-2019-381</a>
+Waliser, Duane, Peter J. Gleckler, Robert Ferraro, Karl E. Taylor, Sasha Ames, James Biard, Michael G. Bosilovich, Otis Brown, Helen Chepfer, Luca Cinquini, <b>Paul J. Durack</b>, Veronika Eyring, Pierre-Phillipe Mathieu, Tsengdar Lee, Simon Pinnock, Gerald L. Potter, Michel Rixen, Roger Saunders, Jorg Shulz, Jean-Noel Thepaut and Matthia Tuma (2020) Observations for Model Intercomparison Project (Obs4MIPs): Status for CMIP6, <i>Geoscientific Model Development</i>, <b>in discussion</b>. DOI: <a href="links/Waliseretal20GMD.html" target="_blank">10.5194/gmd-2019-268</a>
+<div data-badge-popover="right" data-badge-type="2" data-doi="10.5194/gmd-2019-268" data-hide-no-mentions="true" class="altmetric-embed"></div><br>
+Rieger, Landon A., Jason N. Cole, John C. Fyfe, Stephen Po-Chedley, Philip J. Cameron-Smith, <b>Paul J. Durack</b>, Nathan P. Gillett and Qi Tang (2020) Quantifying CanESM5 and EAMv1 sensitivities to volcanic forcing for the CMIP6 historical experiment, <i>Geoscientific Model Development</i>, <b>in discussion</b>. DOI: <a href="links/Riegeretal20GMD.html" target="_blank">10.5194/gmd-2019-381</a>
 <div data-badge-popover="right" data-badge-type="2" data-doi="10.5194/gmd-2019-381" data-hide-no-mentions="true" class="altmetric-embed"></div><br>
-Silvy, Yona, Eric Guilyardi, Jean-Baptiste Sallee and <b>Paul J. Durack</b> (2019) Human-induced changes to the global ocean water-masses and their time of emergence, <i>Nature Climate Change</i>, <b>in review</b>. DOI: <a href="links/Silvyetal19NATClimChg.html" target="_blank">10.1038/xxxx</a>
+Silvy, Yona, Eric Guilyardi, Jean-Baptiste Sallee and <b>Paul J. Durack</b> (2020) Human-induced changes to the global ocean water-masses and their time of emergence, <i>Nature Climate Change</i>, <b>in review</b>. DOI: <a href="links/Silvyetal20NATClimChg.html" target="_blank">10.1038/xxxx</a>
```

Comment:
Silvyetal20NATClimChg.html goes to a "DOI Not Fould" page.

Reply:
@mauzey1 the missing DOI has now been replaced with https://www.nature.com/nclimate/ so should solve this issue